src/main/twirl/uk/gov/hmrc/play/views/helpers/dateFieldsContent.scala.html

```diff
@@ -0,0 +1,55 @@
+@*
```

Comment:
maybe a more conventient name for this? something to say that it includes input fields for all fields